lib/service/errors.ts

```diff
@@ -15,6 +16,10 @@ const errors = {
     message: 'nomatching mode is required',
     code: errorCodes.NOMATCHING_MODE_IS_REQUIRED,
   }),
+  CONNECTION_CLOSED: (rejectionMsg: string) => ({
```

Comment:
Having an error for this is a good idea, but we should also add it to the switch statement in the `getGrpcError` method in GrpcService.ts. I figure this should probably go in the list of cases for `FAILED_PRECONDITION`.

Reply:
Good point!